study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/service/StudyActiveTasksServiceImpl.java

```diff
@@ -322,9 +322,7 @@ public ActiveTaskBo saveOrUpdateActiveTask(ActiveTaskBo activeTaskBo, String cus
         if (activeTaskBo.getScheduleType() != null) {
           addActiveTaskeBo.setScheduleType(activeTaskBo.getScheduleType());
         }
-        if (activeTaskBo.getAnchorDateId() != null) {
-          addActiveTaskeBo.setAnchorDateId(activeTaskBo.getAnchorDateId());
-        }
+        addActiveTaskeBo.setAnchorDateId(activeTaskBo.getAnchorDateId());
```

Comment:
Please update file copyright to add Google LLC & `2020-2021`

Reply:
Updated copyright.